deep_quoridor/src/agents/alphazero/alphazero.py

```diff
@@ -202,7 +230,7 @@ def get_action(self, observation) -> int:
             raise RuntimeError("No nodes visited during MCTS")
         else:
             visit_probs = visit_counts / visit_counts_sum
-            if self.temperature != 1.0:
+            if self.temperature != 0.0:
```

Comment:
I was getting a division by 0 error when playing in the line below, because during play the temperature is set to 0.  Actually I think that if the temperature is 0 we should also be using argmax below instead of np.random.choice, I'll look more into it

Reply:
Yeah, that seems to be caused by the network trying to achieve zeros at the output of the network, which requires an infinite value before the softmax layer.